README.md

```
@@ -44,47 +46,77 @@ After installation, you need to set up your environment variables.
+### Running Specific Test Files
 
 You can run specific test files or suites by providing their path:
 
 ```bash
-npx playwright test tests/aam/authentication/login.spec.js
+npx playwright test tests/aam/login.spec.js
 ```
 
 ## Project Structure
 
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)